example/disasm/full.py

```diff
@@ -142,15 +142,13 @@
 
 
 # Generate dotty graph
-all_blocs = []
+all_blocs = BasicBlocks()
```

Comment:
Actually, "bloc" is the French word for "block". This is an error which is repeated all along Miasm, even in main structure (`asm_bloc`) or module name (`miasm2.core.asmbloc`).
Instead of breaking a lot of APIs, I prefer correct them inside function when I modify code around it. But we may correct it once and for all, a day.

So, following this reasoning, I will fix this one :smile: